geomesa-accumulo/geomesa-accumulo-datastore/src/test/scala/org/locationtech/geomesa/accumulo/index/AttributeIndexStrategyTest.scala

```diff
@@ -590,6 +590,52 @@ class AttributeIndexStrategyTest extends Specification with TestWithDataStore {
       features must haveLength(3)
       features must contain("alice", "bill", "bob")
     }
+
+    "correctly query on indexed attributes with nonsensical AND queries" >> {
+      "redundant int query" >> {
+        val features = execute("age > 25 AND age > 15")
+        features must haveLength(1)
+        features must contain("bob")
+      }
+
+      "int query that returns nothing" >> {
```

Comment:
Sigh.  I could have sworn I saw that label on a test that was doing date filtering.  The longer I stay this afternoon, the more I realize I'm still fogged.  Sorry for the silly question.